i18n/en.json

```diff
@@ -2713,7 +2713,7 @@
   "login.verified": " Email Verified",
   "manage_channel_groups_modal.search_placeholder": "Search groups",
   "manage_team_groups_modal.search_placeholder": "Search groups",
-  "marketplace_modal.install_plugins": "Install Plugins",
+  "marketplace_modal.install_plugins": "Install Plug-ins",
```

Comment:
Question: "Plugin" is used throughout the docs, website etc. Would this change require those instances to also be changed?

Reply:
It would, yes.
I am referring to the Merriam-Webster dictionary for spellings; 'plug-in' is listed there but not 'plugin'.

In fact, any changes that we make to the in-product text will, ideally, have to be reflected in the documentation  wherever the docs have references to menus, buttons etc. I am thinking about an automatic method to make this task less painful, probably by using a transclusion mechanism for UI text used in docs but I haven't completely thought through this idea yet and have only a bare skeleton in my mind. It would need a separate discussion thread, though :)

Reply:
That makes sense - I suspected that "plugin" isn't officially accepted but it's common enough that my browser spellcheck doesn't underline it, hence the question. :) 

That's definitely a separate thread, so happy to do that when the time is right rather than derail this PR. :)

Reply:
@AninditaBasu If I my chime in: `Plugin` is a term very commonly used in the Product. Could you elaborate why it should be changed? To be clear, changing this is an enormous effort both in terms of work and of culture change.

Reply:
Hi @hanzei,

Thank you so much for asking this question 🙂 We'll have similar questions sooner (rather than later). Let me try to respond.

 _Any_  change to in-product text will almost always need:

- Updates to documents because menu commands, options, field names etc. are mentioned so often in the docs.
- Updates to the corresponding non-English-language string files.

Keeping this is mind, the next question is: should we do updates related to word preferences and spellings when one word or phrase is almost equally well-known and used as the another word or phrase. Me, I like things to be consistent, so if I (as a user) see `plug-in` at one place and `plugin` at another, I would (at best) ignore and move on or (at worst) if I am a non-English speaker still coming to grips with tech-world idioms in English, puzzled and wonder if the two words mean two different things. I value consistency, and while I myself have no preference for one word over another when both convery the same meaning, I tend to defer to standards such as dictionaries (Merriam-Webster in this case), where `plug-in` is listed but not `plugin`.

Um, am I making sense? 🤔

Reply:
That does make sense and I agree with your approach. In this case though, I would like to suggest we retain "plugin" because it's what we use in the UI, all the documentation, and channel names. 

I'd say that the instances of "plug-in" would be easier to remedy than changing all instances of "plugin" to "plug-in". I did a very quick Google comparison and "plugin" is used in Google Chrome and WordPress  and comes up as a thing you download and use, whereas "plug-in" is just a list of definitions.  There is, of course, debate around this especially as the MSTP uses "plug-in" a lot of companies use that as the basis for the in-house style guide. 

I'm 5/5 for keeping it as "plugin" as it very widely-used and understood. As a random side-note, "plugin" isn't picked up by my browser spellcheck as incorrect so anecdotally I think that is something to consider. 

We also don't really use hyphens in any of our other UI elements so I think it would be visually jarring to introduce it. @michaelgamble would also be a good person for input on this.

Reply:
To add one data point: There are currently only seven occurrences of `plug-in` in the docs and most of the don't reference the plugin system and instead some other kind of software.

Hence, I would argue that we are quite consistent currently.

Reply:
Can someone please clarify ...  according to this PR we would be changing 'Plugin' to 'Plug-in' in the navbar dropdown. However the discussion heretofore concludes that we should stick with 'Plugin'. Am i right?